### PR TITLE
Add end delay to TimingDict

### PR DESCRIPTION
**Pull request by shans**

### CODE REVIEW

**Comment by dstoc** [comment]
lgtm, but we should probably test how the effect is applied during the endDelay

File: web-animations.js [hunk 1]
```diff
@@ -494,7 +495,7 @@ TimedItem.prototype = {
     return result;
   },
   get endTime() {
-    return this._startTime + this.activeDuration + this.specified.delay;
+    return this._startTime + this.activeDuration + this.specified.delay + this.specified.endDelay;
```
Comment (dstoc) [inline web-animations.js]: need to wrap this line

File: web-animations.js [hunk 2]
```diff
@@ -1300,8 +1301,8 @@ TimingGroup.prototype = createObject(TimedItem.prototype, {
       } else if (this.type === 'seq') {
         var result = 0;
         this._children.forEach(function(a) {
-          result += a.activeDuration + a.specified.delay;
-        });
+          result += a.activeDuration + a.specified.delay + a.specified.endDelay;
+      });
```
Comment (dstoc) [inline web-animations.js]: indentation is wrong

**Comment by dstoc** [comment]
lgtm

File: web-animations.js (hunk 1 above)
Comment (shans) [inline web-animations.js]: Done.

File: web-animations.js (hunk 2 above)
Comment (shans) [inline web-animations.js]: Done.

**Pull request merged**